README.md

```diff
@@ -40,6 +35,19 @@ How-to pages from this repo:
 
 Note that [official releases][] are [published to Maven Central][].
 
+## Upgrading
+
+### 1.0.0 -> 1.1.0
+
+The `grpc-kotlin-stub-lite` library no longer exists as `grpc-kotlin-stub` no longer has a dependency on
+`grpc-protobuf` or `protobuf-java-util`.  But this means you now need to specify those or the lite dependencies when
+you use `grpc-kotlin-stub`.  For examples, see: [examples/stub/build.gradle.kts](examples/stub/build.gradle.kts),
+[examples/stub-lite/build.gradle.kts](examples/stub-lite/build.gradle.kts), or
+[examples/stub-android/build.gradle.kts](examples/stub-android/build.gradle.kts).
+
+The `javax.annotation:javax.annotation-api` dependency is now transitive so you do not need to specify it manually.
```

Comment:
I think that we can offer a smoother read. How about something like the following:

---

- The `grpc-kotlin-stub-lite` library no longer exists, instead use  
  `grpc-kotlin-stub`. For details, see the next point.
- The `grpc-kotlin-stub` library no longer depends on `grpc-protobuf` or. 
  `protobuf-java-util`, so your project will need to include protobuf. 
  dependencies itself. For examples, see. 
  [examples/stub/build.gradle.kts](examples/stub/build.gradle.kts),  
  [examples/stub-lite/build.gradle.kts](examples/stub-lite/build.gradle.kts), or. 
  [examples/stub-android/build.gradle.kts](examples/stub-android/build.gradle.kts).
- `grpc-kotlin-stub` now transitively includes
  `javax.annotation:javax.annotation-api`, so you can drop explicit dependencies
  from your project's build spec.

---

I'm not sure how relevant it is for the reader to know about "transitive inclusion", so the last bullet could (maybe) more simply be:

- `grpc-kotlin-stub` now exports `javax.annotation:javax.annotation-api`, so you
  can drop it from your project's explicitly listed dependencies.